src/ugc.rs

```diff
@@ -9,6 +9,8 @@ use std::mem;
 use std::path::Path;
 use std::os::raw::c_char;
 
+pub use sys::kNumUGCResultsPerPage as kNumUGCResultsPerPage;
```

Comment:
Should follow [rust style guidelines](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/advice.md#names). Also should use rust types for the public facing API.

e.g.
```suggestion
pub const RESULTS_PER_A_PAGE: u32 = sys::kNumUGCResultsPerPage as u32;
```

Reply:
What does the `_A_` signify? I think `RESULTS_PER_PAGE` is probably sufficient.

Reply:
Just thought it read a bit better but I don't mind about the name too much.